packages/marketplace/contracts/TransferManager.sol

```diff
@@ -129,6 +141,16 @@ abstract contract TransferManager is Initializable, ITransferManager {
         emit ProtocolFeeSet(newProtocolFeePrimary, newProtocolFeeSecondary);
     }
 
+    /// @notice Sets the LAND contract address.
+    /// @param newLandContractAddress Address of new LAND contract
+    function _setLandContract(ILandToken newLandContractAddress) internal {
+        // TODO: uncomment when ILandToken is supported by LandBase
+        // require(ERC165Checker.supportsInterface(address(newLandContractAddress, type(ILandToken).interfaceId), "invalid LAND address");
```

Comment:
TODO when `packages/land` is updated

Reply:
@atkinsonholly Do you remember what is supposed to be updated in `packages/land`?